src/Illuminate/Foundation/Console/StorageLinkCommand.php

```diff
@@ -28,9 +30,10 @@ class StorageLinkCommand extends Command
     public function handle()
     {
         $relative = $this->option('relative');
+        $force = $this->option('force');
 
         foreach ($this->links() as $link => $target) {
-            if (file_exists($link)) {
+            if (file_exists($link) && ! (is_link($link) && $force)) {
```

Comment:
Can you rewrite this conditional? It is hard to read.

Reply:
Is this better?